backuprestore/backup.py

```diff
@@ -168,7 +168,7 @@ def main():
                     iaas_client.exit(
                         'Could not upload the tarball {}.'.format(metadata_files_path))
 
-            if landscape == 'Azure':
+            if landscape == 'Azure' or landscape == 'Gcp':
```

Comment:
Check if Aws needs copy_snapshot/ encryption. If not remove these checks.

Reply:
For AWS it is not needed to copy snapshot if the disk is already encrypted. But since this is a reference service, we can keep the copy_snapshot function here.